### DISCUSSION TITLE
Pastel Powerline preset color theme

### DISCUSSION THREAD

**Discussion by fiqrisr** [starship/starship#4141] [Q&A] [answered]
Anyone know what color theme is this? Pastel powerline preset looks terible on my terminal

![image](https://user-images.githubusercontent.com/30799382/176843321-ce762eb4-7d7b-4047-a86b-3ddbdde9c6e0.png)

My terminal:
![image](https://user-images.githubusercontent.com/30799382/176843282-9fe6e381-8bce-41e4-ba8c-3cb404db51bb.png)

**Comment by davidkna** [comment, marked as answer]
All screenshots should be using the snazzy color theme. It might be linked in the FAQ section on the website.

**Reply by fiqrisr** [reply]
Thanks
